src/main/java/seedu/address/logic/commands/AddJobCommand.java

```diff
@@ -63,7 +63,7 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         model.addJob(toAdd);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd), "Jobs");
+        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd), Job.TAB_NAME);
```

Comment:
Good job changing this from a String to a class variable! Good coding practise, I like!